
# HG changeset patch
# User shade
# Date 1544788619 -3600
# Node ID d4da64e0e9163b70a8cfc950ea7a9d61a8b9ada4
# Parent  33d33996a638f3eb44befe3c21fa8788808dc3e8
8215374: 32-bit build failures after JDK-8181143 (Introduce diagnostic flag to abort VM on too long VM operations)
Reviewed-by: roland

diff -r 33d33996a638 -r d4da64e0e916 src/hotspot/share/runtime/vmThread.cpp
--- a/src/hotspot/share/runtime/vmThread.cpp	Fri Dec 14 12:56:57 2018 +0100
+++ b/src/hotspot/share/runtime/vmThread.cpp	Fri Dec 14 12:56:59 2018 +0100
@@ -206,7 +206,7 @@
   if (is_armed()) {
     jlong delay = (os::javaTimeMillis() - _arm_time);
     if (delay > AbortVMOnVMOperationTimeoutDelay) {
-      fatal("VM operation took too long: " SIZE_FORMAT " ms (timeout: " SIZE_FORMAT " ms)",
+      fatal("VM operation took too long: " JLONG_FORMAT " ms (timeout: " INTX_FORMAT " ms)",
             delay, AbortVMOnVMOperationTimeoutDelay);
     }
   }

